system/modules/core/library/Contao/Database/Mysql.php

```diff
@@ -95,6 +95,8 @@ protected function get_error()
 	 */
 	protected function find_in_set($strKey, $varSet, $blnIsField=false)
 	{
+		$strKey = static::quoteIdentifier($strKey);
+
 		if ($blnIsField)
 		{
 			return "FIND_IN_SET(" . $strKey . ", " . $varSet . ")";
```

Comment:
In this case we should probably use `"FIND_IN_SET(" . $strKey . ", " . static::quoteIdentifier($varSet) . ")";` because `$varSet` is then an identifier too.